lib/tagging_sync_verifier.rb

```diff
@@ -31,22 +32,24 @@ def verify_taggings_are_in_sync
       from_content_store
     )
 
-    if comparison.same?
-      puts "Taggings are in sync!"
+    only_in_content_api = comparison.diffs[:only_a]
+    only_in_content_store = comparison.diffs[:only_b].select { |content_id, _| from_content_api[content_id] }
+
+    if only_in_content_api.empty? && only_in_content_store.empty?
```

Comment:
I wonder if it would be useful to extract out a Diff object from the Comparer rather than using an array. I think the logic was very clear when it was all encapsulated in the Comparer, and now we are actually doing something with the diffs besides printing some of that has to be repeated here.

Also it seems like the Comparer `only_a` `only_b` methods could just be public since they're the actual things you need from it.